GPU available: False, used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/pytorch_lightning/trainer/configuration_validator.py:72: You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.
wandb: Currently logged in as: allepalma. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.15.12
wandb: Run data is saved locally in ../project_folder/experiments/20231107_bbbc021_try/wandb/run-20231107_134355-ya8u43wr
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run cerulean-sun-37
wandb: ‚≠êÔ∏è View project at https://wandb.ai/allepalma/bbbc021_try
wandb: üöÄ View run at https://wandb.ai/allepalma/bbbc021_try/runs/ya8u43wr

  | Name             | Type         | Params
--------------------------------------------------
0 | embedding_matrix | Embedding    | 960   
1 | generator        | DataParallel | 24.3 M
2 | style_encoder    | DataParallel | 14.3 M
3 | discriminator    | DataParallel | 14.3 M
4 | mapping_network  | DataParallel | 11.3 K
--------------------------------------------------
53.0 M    Trainable params
960       Non-trainable params
53.0 M    Total params
211.840   Total estimated model params size (MB)
SLURM auto-requeueing enabled. Setting signal handlers.
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** JOB 14162118 ON gpusrv59 CANCELLED AT 2023-11-07T13:52:37 ***
submitit WARNING (2023-11-07 13:52:37,082) - Bypassing signal SIGCONT
slurmstepd: error: *** STEP 14162118.0 ON gpusrv59 CANCELLED AT 2023-11-07T13:52:37 ***
[rank: 0] Received SIGTERM: 15
Bypassing SIGTERM: 15
submitit WARNING (2023-11-07 13:52:37,084) - Bypassing signal SIGTERM
wandb: Waiting for W&B process to finish... (success).
wandb: 
wandb: Run history:
wandb:       D/latent_fake ‚ñÅ
wandb:       D/latent_real ‚ñÅ
wandb:        D/latent_reg ‚ñÅ
wandb:         G/lambda_ds ‚ñÅ
wandb:        G/latent_adv ‚ñÅ
wandb:        G/latent_cyc ‚ñÅ
wandb:         G/latent_ds ‚ñÅ
wandb:        G/latent_sty ‚ñÅ
wandb:               epoch ‚ñÅ
wandb: trainer/global_step ‚ñÅ
wandb: 
wandb: Run summary:
wandb:       D/latent_fake 1.52056
wandb:       D/latent_real 0.13093
wandb:        D/latent_reg 0.00172
wandb:         G/lambda_ds 0.9999
wandb:        G/latent_adv 0.91632
wandb:        G/latent_cyc 0.8737
wandb:         G/latent_ds 0.60974
wandb:        G/latent_sty 1.09803
wandb:               epoch 0
wandb: trainer/global_step 9
wandb: 
wandb: üöÄ View run cerulean-sun-37 at: https://wandb.ai/allepalma/bbbc021_try/runs/ya8u43wr
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ../project_folder/experiments/20231107_bbbc021_try/wandb/run-20231107_134355-ya8u43wr/logs
Exception in thread IntMsgThr:
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/threading.py", line 980, in _bootstrap_inner
Exception in thread ChkStopThr:
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/threading.py", line 980, in _bootstrap_inner
    self.run()
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/threading.py", line 917, in run
self.run()
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 285, in check_stop_status
    self._target(*self._args, **self._kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 299, in check_internal_messages
    self._loop_check_status(
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 223, in _loop_check_status
self._loop_check_status(
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 223, in _loop_check_status
    local_handle = request()
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 743, in deliver_internal_messages
    local_handle = request()
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 727, in deliver_stop_status
    return self._deliver_internal_messages(internal_message)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 481, in _deliver_internal_messages
return self._deliver_stop_status(status)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 459, in _deliver_stop_status
    return self._deliver_record(record)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 428, in _deliver_record
return self._deliver_record(record)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 428, in _deliver_record
    handle = mailbox._deliver_record(record, interface=self)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
handle = mailbox._deliver_record(record, interface=self)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 455, in _deliver_record
    interface._publish(record)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
interface._publish(record)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/interface/interface_sock.py", line 51, in _publish
    self._sock_client.send_record_publish(record)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
self._sock_client.send_record_publish(record)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 221, in send_record_publish
    self.send_server_request(server_req)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self.send_server_request(server_req)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 155, in send_server_request
    self._send_message(msg)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._send_message(msg)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 152, in _send_message
    self._sendall_with_error_handle(header + data)
      File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
self._sendall_with_error_handle(header + data)
  File "/home/icb/alessandro.palma/miniconda3/envs/IMPA/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)    
sent = self._sock.send(data)BrokenPipeError
: BrokenPipeError[Errno 32] Broken pipe: 
[Errno 32] Broken pipe
